server/src/main/java/mateusz/michal/chat/Registration/Service/RegistrationService.java

```diff
@@ -123,37 +119,41 @@ private User createUser(UserDTO userDTO){
 
     private List<MyError> validateUserPassword(String password){
         List<MyError> errors = new ArrayList<>();
-        if(isPasswordNull(password)){
+        if(isNull(password)){
             errors.add(new MyError(400, RegisterFormErrorCode.PASSWORD_NULL,
                     "parameter password is null"));
         } else {
-            if(isPassworEmptyString(password)){
+            if(isEmpty(password)){
                 errors.add(new MyError(422,RegisterFormErrorCode.PASSWORD_MISSING,
                         "parameter password isn't present"));
             }
-            if(isPasswordIncorrect(password)){
+            if(!isPasswordStrong(password)){
                 errors.add(new MyError(422,RegisterFormErrorCode.WEAK_PASSWORD,
                         "password is too weak"));
             }
+            if(isPasswordToShort(password)){
+                errors.add(new MyError(422,RegisterFormErrorCode.SHORT_PASSWORD,
+                        "password is too short"));
+            }
+            if(isPassworToLong(password)){
+                errors.add(new MyError(422,RegisterFormErrorCode.LONG_PASSWORD,
+                        "password is too long"));
+            }
         }
         return errors;
     }
 
     // methods that validate single elements of properties
-    private boolean isNameEmptyString(String name){
-        return name.equals("");
-    }
-
-    private boolean isNameIncorrect(String name){
-        return (name.startsWith(" ") || name.endsWith(" ") || hasNameSpecialCharacters(name));
+    private boolean isEmpty(String property){
+        return property.equals("");
     }
 
-    private boolean isEmailEmptyString(String email){
-        return email.equals("");
+    private boolean isNull(String property){
+        return property == null;
     }
 
-    private boolean isPassworEmptyString(String password){
-        return password.equals("");
+    private boolean isNameIncorrect(String name){
```

Comment:
nit: teoretycznie większy sens miałoby trimowanie stringa przy otrzymaniu go (usuwanie leading i trailing spaces), wtedy trafiałby od razu walidatora `isEmpty`, dzięki czemu `isNameIncorrect` nie musiałoby istnieć, ale to tylko inny pomysł :)

Reply:
isNameIncorrect poza trimowaniem sprawdza również czy w imieniu nie występują znaki specjalne. 
Jednak zmienię nazwę metody na IsNameCorrect (tak jak zostało to zrobione z Emailem)